src/View/Card/show.html.twig

```diff
@@ -2,9 +2,15 @@
 
 <h1 class="unlock">UNLOCK !</h1>
 <h2 class="unlock-title2">Sherlock Adventure</h2>
-<div class="car-show">
-    <h2 class="card-name">{{ card.name }}</h2>
-    <img alt="image de la carte">
-    <p class="card-description">{{ card.description }}</p>
+<div class="card-container">
+    <div class="card-show">
+        <div class="card-head">
+            <img src="/assets/images/card-icon.png" alt="numéro de carte" width="60rem">
```

Comment:
you already have a size on card-show in css, so you can remove width from you img tag :)